src/main.cpp

```diff
@@ -99,6 +99,8 @@ int main() {
     }
   }
 
+  chThdSleepMilliseconds(1000);
+
```

Comment:
we could have a check, if debug build + add comment as to why its here (gives time for debugger to attach)